compose/bin/debug-cli

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+S=$(bin/clinotty cat /usr/local/etc/php/php.ini | grep -iGc 'xdebug.mode = off');
+
+if [[ $S == 1 ]]; then
+  echo "Enable xdebug first"
```

Comment:
Why not run `bin/xdebug enable` if you've determined it needs to be done? You can remove the check entirely since `bin/xdebug` itself will detect it's enabled?

Reply:
@h3xx yes of course, maybe could be also as an additional argument of bin/xdebug like --cli

Reply:
@massalinux I think he means instead of running the line:

    echo "Enable xdebug first"

Just run

    bin/xdebug enable

This way rather than prompting the user to enable Xdebug, you're just enabling it.

Reply:
I implied removing lines 2-5 entirely including the check (`S=$(...)`) and replace with something like:
```sh
# Ensure xdebug is enabled
bin/xdebug enable
```

Reply:
yes of course, what i meant was that you can actually incorporate this functionality directly inside the bin/xdebug command as on optional parameter instead of create a new command.

Meanwhile i've modified as you suggested the command. I'll add the suggestion from @markshust and update the PR.